app/views/tools/_petition.html.erb

```diff
@@ -104,7 +104,9 @@
     </div>
   </div>
 </div>
-
+</div>
+</div>
+</div>
```

Comment:
What's the reason for these additional closing div tags? It looks like they might not have matching opening tags in the template.

Reply:
That was a hack I forgot to remove, removing now.